samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -99,7 +100,7 @@ Buttons:
    All buttons have the same functionality during the provisioning procedure.
 
 Button 1:
-   Simulates presence detected (after the provisioning procedure is finished).
+   Simulates presence detected (after the provisioning procedure is finished). The time before the presence is detected depends on the motion threshold.
```

Comment:
Should we add that (as I understand) the button needs to be pressed the required time to send presence detection?